README.md

```diff
@@ -8,46 +8,58 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 Clone the repo and cd into it
 
-```
+```bash
 git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`)
+<!-- Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`) -->
 
-```
+Now, obtain the `.env` file for the environment (ask someone on the engineering team) and save it.
+
+<!-- ```
 # copy in the .env files
 # symlink the staging .env file
 ln -s .env.staging.local .env
-```
+``` -->
 
-Now you're ready to start the server
+Install the npm modules
```

Comment:
```suggestion
Install the platform dependencies with `npm`:

(**Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions)
```